test/NameBinding/Inputs/NamedLazyMembers/NamedLazyMembers.h

```diff
@@ -0,0 +1,33 @@
+#import <Foundation/Foundation.h>
+
+@protocol doer
+
+- (void) doSomeWork;
+
+- (void) doSomeWorkWithSpeed:(int)s;
+
+- (void) doSomeWorkWithSpeed:(int)s andThoroughness:(int)t
+  NS_SWIFT_NAME(doSomeWork(speed:thoroughness:));
+
+// These we are generally trying to not-import, via laziness.
+- (void) goForWalk;
+- (void) takeNap;
+- (void) eatMeal;
+- (void) tidyHome;
+- (void) callFamily;
+- (void) singSong;
+- (void) readBook;
+- (void) attendLecture;
+- (void) writeLetter;
+
+@end
+
+
+@interface simpleDoer
+
++ (simpleDoer*) doer;
+
++ (simpleDoer*) doerOfNoWork;
```

Comment:
What are you testing with this? Was this class supposed to conform to the protocol you defined above?

Reply:
No (doing so actually causes the protocol to be entirely-imported because presumably conformance checking). I was hoping to add some members that could possibly (wrongly) collide in the member lookup table with the members we _do_ want, to be sure we don't accidentally grab them. Maybe there's no chance of that happening.